R/iSEE-extras.R

```diff
@@ -788,15 +788,15 @@ height_limits <- c(400L, 1000L)
     # Destroy all non-atomic fields (only internal, no need to hold commands).
     for (f in colnames(rowData(tmp_se))) {
         cur_field <- rowData(tmp_se)[[f]]
-        if (!is.numeric(cur_field) && !is.factor(cur_field) 
```

Comment:
Wait, what? Why? `&&` is more efficient, it skips evaluation of the second field when the first field is false.

Reply:
Oh, good, you were reverting it.